observability/otel/otel-rn.adoc

```diff
@@ -12,6 +12,73 @@ You can use the {OTELName} xref:otel-forwarding-telemetry-data.adoc#otel-forward
 
 include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
 
+[id="otel_3-3-1_{context}"]
+== Release notes for {OTELName} 3.3.1
+
+The {OTELName} is provided through the {OTELOperator}.
+
+The {OTELName} 3.3.1 is based on the open source link:https://opentelemetry.io/docs/collector/[OpenTelemetry] release 0.107.0.
```

Comment:
@pavolloffay is the upstream version the correct one? Because the change was included in 0.108.0 if I'm not mistaken https://github.com/open-telemetry/opentelemetry-operator/pull/3214

Reply:
It is correct https://github.com/os-observability/opentelemetry-operator/tree/release-v0.107